pkg/kubelet/network/cni/cni.go

```diff
@@ -266,6 +266,9 @@ func (plugin *cniNetworkPlugin) GetPodNetworkStatus(namespace string, name strin
 	if err != nil {
 		return nil, fmt.Errorf("CNI failed to retrieve network namespace path: %v", err)
 	}
+	if netnsPath == "" {
+		return nil, fmt.Errorf("Cannot find network namespace for the terminated container %q", id)
```

Comment:
Finally we will get two same log info with different log levels, right? So I think we'd better modify the error info here.